.github/workflows/macos-build.yml

```diff
@@ -168,23 +187,27 @@ jobs:
           --name $DEPSHASH \
           --debug
   build:
-    runs-on: macos-13-xlarge
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [ macos-13-large, macos-14-xlarge ]
```

Comment:
We should probably add a clarifying note here that `macos-14-xlarge` is arm and `macos-13-large` is x64. Beyond that - it's my understanding per https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#about-macos-larger-runners that `macos-14-large` is x64 too, should we maybe use that for consistency?

Reply:
Yes, and good callout - added both the comment and changed the Intel runner to macos-14-large